build/update-locales.sh

```diff
@@ -9,11 +9,11 @@ fi
 mv ./options/locale/locale_en-US.json ./options/
 
 # Remove translation under 25% of en_us
-baselines=$(wc -l "./options/locale_en-US.json" | cut -d" " -f1)
+baselines=$(wc -l "./options/locale_en-US.json" | awk '{print $1}')
```

Comment:
```suggestion
baselines=$(cat "./options/locale_en-US.json" | wc -l)
```

`wc` will not output a filename when input is stdin.